ui/k-info-panel/link_info.kytos

```diff
@@ -177,6 +210,42 @@
          _this.$kytos.$emit("setNotification", notification)
        });
      },
+     bt_delete(){
+      this.show_modal = true;
+     },
+     modal_cancel(){
+      this.show_modal = false;
+     },
+     modal_delete(){
+      this.show_modal = false;
+      this.delete_link();
+     },
+     delete_link(){
+      let name = this.metadata.link_name !== undefined && this.metadata.link_name.length !== 0? '"' + this.metadata.link_name + '"' : this.details.id
```

Comment:
![20240129_093448](https://github.com/kytos-ng/topology/assets/1010796/8be2da6b-c31c-413a-b0f5-296d5befebbd)

We might need to truncate the `link.id` as a substring since it can get a bit length based on the component size. Also, check out the message string order `Delete Link <id>?` would read more naturally.

Reply:
On my end the box is enlarged. 
![Screenshot_20240129_081050](https://github.com/kytos-ng/topology/assets/55767214/36a6e0ec-fc45-4846-b1ff-1a24584c670c)
The cache could be the culprit. If not, what browser are you using?

Reply:
I've just tried with new incognito tabs and same behavior. I tried on `google-chrome` v116 and `brave` v121, still seeing the text going over the component, interesting: 

![20240129_102302](https://github.com/kytos-ng/topology/assets/1010796/0f38e44d-e47d-41d3-987d-171dc02e3a01)

![20240129_102357](https://github.com/kytos-ng/topology/assets/1010796/04dc782c-5d34-4f70-8311-b61398753a27)

Reply:
Deleted fixed `width`.

Reply:
Great. Now it's working for me too. Let's close this thread.